src/Frontend/Page/Delegation.hs

```diff
@@ -130,19 +130,21 @@ topicDelegation tid = formPageHandlerCalcMsgM
     pageDelegateVoteSuccessMsg
 
 -- | 13. Delegation network
-data PageDelegationNetwork = PageDelegationNetwork DScope DScopeTree DelegationNetwork
+data PageDelegationNetwork = PageDelegationNetwork DScope DScopeForest DelegationNetwork
   deriving (Eq, Show, Read)
 
-newtype DScopeTree = DScopeTree (Tree DScopeFull)
+newtype DScopeForest = DScopeForest [Tree DScopeFull]
   deriving (Eq, Show, Read)
 
-instance Aeson.ToJSON DScopeTree where
-    toJSON (DScopeTree t) = f t
+-- TODO: Fix JSON instance
+instance Aeson.ToJSON DScopeForest where
+    toJSON (DScopeForest ts) = f ts
       where
-        f (Tree.Node dscope chldrn) = Aeson.object
+        f [] = error "impossible"
```

Comment:
that's not really impossible, though, right?  isn't it a legitimate corner case to have no dscopes and no delegations?